admin/mod_list.php

```diff
@@ -0,0 +1,221 @@
+                  <?php echo HEADING_PAGE_MENU_KEY; ?><br>
+                </td>
+                <td class="dataTableHeadingContent" align="left" valign="top">
+                  <?php echo HEADING_DISPLAY; ?><br>
+                </td>
+                <td class="dataTableHeadingContent" align="left" valign="top">
+                  <?php echo HEADING_PAGE_LINK; ?><br>
+                </td>
+              </tr>
+              <!-- end heading row -->
+<?php
+    $new_pages = array(); 
+    $pages_query_raw = " SELECT * FROM " . TABLE_ADMIN_PAGES; 
+    $pages = $db->Execute($pages_query_raw); 
+    if ($pages->RecordCount() <= 0) { 
+?>
+      <tr><td colspan="3" align="left"><?php echo '<b>' . NO_PAGES_TABLE_FOUND . '</b>'; ?></td></tr>
+<?php
+    } else { 
+       $unknown_pages = 0; 
+       while (!$pages->EOF) {
+          $key = $pages->fields['language_key']; 
+          if (in_array($key, $built_in_boxes)) { 
+             $pages->MoveNext();
+             continue;
+          }
+          $unknown_pages++; 
+   ?>
+                <tr>
+                   <td class="dataTableContent" align="left">
+                   <?php 
+                      if (defined($pages->fields['language_key'])) 
+                         echo constant($pages->fields['language_key']);
+                      else 
+                         echo "(" . $pages->fields['language_key'] . ")";
+                   ?>
+                   </td>
+                   <td class="dataTableContent">
+   <?php 
+                         echo $pages->fields['menu_key'];
+   ?>
+                   </td>
+                   <td class="dataTableContent">
+   <?php 
+                         echo $pages->fields['display_on_menu'];
+   ?>
+                   </td>
+                   <td class="dataTableContent">
+                   <?php 
+                      if (defined($pages->fields['language_key']) && 
+                          defined($pages->fields['main_page'])) {
+                         echo '<a href="' . zen_href_link(constant($pages->fields['main_page']), $pages->fields['page_params']) .'">' . constant($pages->fields['language_key']) . '</a>';  
+                      } else {
+                         echo NO_LINK; 
+                      }
+                   ?>
+                   </td>
+                 </tr>
+   <?php
+         $pages->MoveNext();
+       }
+       if ($unknown_pages == 0) { 
+?>
+          <tr>
+            <td colspan="3"><?php echo NO_NEW_PAGES; ?></td>
+          </tr>
+<?php
+       }
+    }
+?>
+</table>
+
+<h2><?php echo DB_LIST; ?></h2>
+<?php
+    $new_pages = array(); 
+    $tables_query_raw = "SELECT TABLE_NAME from INFORMATION_SCHEMA.TABLES where TABLE_SCHEMA = '" . DB_DATABASE . "'"; 
+    $tables = $db->Execute($tables_query_raw); 
+    if ($tables->RecordCount() <= 0) { 
+?>
+      <tr><td colspan="3" align="left"><?php echo '<b>' . NO_tables_FOUND . '</b>'; ?></td></tr>
```

Comment:
`NO_tables_FOUND` ... should this be upper-case, and have a lang-define to match it?